include/alpaka/mem/view/Accessor.hpp

```diff
@@ -23,7 +23,7 @@
 
 #include <tuple>
 
-namespace alpaka::experimental
```

Comment:
Please check that you move only SYCL stuff out of the experimental namespace.

Reply:
As far as I can tell, this Accessor file is included only in SYCL (buffer and accessor). I have not found a way to leave these files inside the experimental namespace without breaking SYCL

Reply:
Yes, I built the accessors a long time ago to prepare for the SYCL backend. But when we move to pointer based SYCL, they are no longer needed. Except for the use cases of @psychocoderHPC who wants to configure how memory access works, but this can also be handled by `std::mdspan`. I want to drop accessors in a separate PR, see also #1897.